cfgov/v1/models/organisms.py

```diff
@@ -13,6 +14,29 @@ class Meta:
         template = '_includes/organisms/well.html'
 
 
+class FullWidthText(blocks.StructBlock):
+    content = blocks.RichTextBlock(required=True)
+
+    class Meta:
+        icon = 'title'
+        template = '_includes/organisms/full-width-text.html'
+
+
+class PostPreview(blocks.StructBlock):
+    heading = blocks.CharBlock(max_length=100, required=True)
+    body = blocks.RichTextBlock(required=True)
+    image = atoms.ImageBasic(required=False)
+
+    post = blocks.PageChooserBlock(required=True)
+
+    comments_close_date = blocks.DateTimeBlock(required=False)
```

Comment:
@KimberlyMunoz @duelj comments_close_date will people be able to comment on all posts? or will this is be post type specific. Currently i'm grabbing post information from the posts themselves (event data, published date etc.). I think comments_close_date could retrieved from the post too?